packages/twenty-front/src/modules/workflow/components/WorkflowStepDetail.tsx

```diff
@@ -94,6 +96,19 @@ export const WorkflowStepDetail = ({
             />
           );
         }
+        case 'RECORD_CRUD': {
+          if (isWorkflowRecordCreateAction(stepDefinition.definition)) {
+            return (
+              <WorkflowEditActionFormRecordCreate
+                action={stepDefinition.definition}
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                {...props}
```

Comment:
think we should avoid props spreading. It lower the readability of the code
just pass all 
```suggestion
      stepId={stepId}
      workflowVersion={workflowVersion}
      readonly={readonly}
      onTriggerUpdate={onTriggerUpdate}
      onActionUpdate={onActionUpdate}
```

Reply:
Here, the `props` object is a type union. If I provide the properties of this object individually, I will loose the type union and, for instance, `props.readonly` will be a boolean.

![CleanShot 2024-11-18 at 15 01 41@2x](https://github.com/user-attachments/assets/fb1b3461-5686-45e7-84db-88f92a0f8cb6)

Reply:
This component serves as a dispatcher. I think it's better to have strong types than to provide props as usually recommended.

Another way to keep the benefits without disabling the ESLint rule would be to have a single prop containing `readonly`, `onTriggerUpdate`, `onActionUpdate`:

```tsx
<WorkflowStepDetail
  xxx={{ readonly: true }}
/>

<WorkflowStepDetail
  xxx={{ onActionUpdate: yyy, onTriggerUpdate: zzz }}
/>
```

Reply:
Done 👌